src/AdvancedVI.jl

```diff
@@ -38,7 +39,14 @@ Evaluate the value and gradient of a function `f` at `x` using the automatic dif
 - `aux`: Auxiliary input passed to `f`.
 - `out::DiffResults.MutableDiffResult`: Buffer to contain the output gradient and function value.
 """
-function value_and_gradient! end
+function value_and_gradient!(
```

Comment:
This might be better as an extension x/ref https://github.com/gdalle/DifferentiationInterface.jl/issues/509